google/iam/admin/v1/iam_gapic.yaml

```diff
@@ -0,0 +1,248 @@
+type: com.google.api.codegen.ConfigProto
+language_settings:
+  java:
+    package_name: com.google.cloud.iam.admin.spi.v1
+  python:
+    package_name: google.cloud.iam.admin.v1
+  go:
+    package_name: cloud.google.com/go/iam/admin/apiv1
+  csharp:
+    package_name: Google.Iam.Admin.V1
+  ruby:
+    package_name: Google::Cloud::Iam::Admin::V1
+  php:
+    package_name: Google\Cloud\Iam\Admin\V1
+  nodejs:
+    package_name: '@google-cloud/admin'
```

Comment:
Shouldn't this be `@google-cloud/iam-admin` or `@google-cloud/iam/admin`, to be consistent with how we're namespacing the other languages to avoid conflict with the mixin API? Or is NodeJS somehow avoiding that conflict?